server/auth.js

```diff
@@ -42,6 +42,19 @@ internals.applyStrategy = function (server, next) {
                     }
 
                     done(null, Object.keys(results.user.roles));
+                }],
+                updateSession: ['scope', function (results, done) {
+
+                    if (!results.roles) {
+                        return done();
+                    }
+
+                    const update = {
+                        $set: {
+                            lastLogin: new Date()
+                        }
+                    };
+                    Session.findByIdAndUpdate(results.session._id.toString(),update, done);
```

Comment:
This auth strategy runs for every request, not just during a login event. So I'm not sure that updating the `lastLogin` field of the session record is appropriate. If this updated something like `lastActive` that would make more sense. What was the intent?

Reply:
Yes, that was the intent, `lastActive` is a better term